src/community/gwc-azure-blob/src/main/resources/applicationContext.xml

```diff
@@ -11,7 +11,7 @@
    Bean configuration file for the gwc-azure-blob module
   </description>
   
-  <bean id="AzureBlobStoreConfigProvider" class="org.geowebcache.azure.AzureBlobStoreConfigProvider" depends-on="geoWebCacheExtensions">
```

Comment:
Are you sure? Hmm.. looking at the code, indeed it does not seem like it's needed.